### DISCUSSION TITLE
show the key mapping on the screen

### DISCUSSION THREAD

**Discussion by hsain9357** [barry-ran/QtScrcpy#998] [Q&A] [answered]
I want to show the key mapping on the screen when playing games so i can know what key should i press for a specific job just like the picture from the readme file of the repo

![game](https://github.com/barry-ran/QtScrcpy/assets/90194836/7b32378a-5887-4eff-ae52-c9ec38e0330e)

**Comment by DB-BOY** [comment]
https://github.com/barry-ran/QtScrcpy/blob/dev/docs/KeyMapDes.md

**Reply by hsain9357** [reply]
I can't find a way to show key mapping buttons on the screen with this URL  instructions

**Reply by end3r-man** [reply]
> I can't find a way to show key mapping buttons on the screen with this URL instructions

any solutions ??

**Reply by hsain9357** [reply]
> > I can't find a way to show key mapping buttons on the screen with this URL instructions
> 
> any solutions ??

didn't find a solution .

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper

**Comment by barry-ran** [comment, marked as answer]
https://lrbnfell4p.feishu.cn/docx/YLCmd8rN5oiK1axvjD0cGgjhnOh
